.github/workflows/vulnerability-scan.yml

```diff
@@ -0,0 +1,54 @@
+name: Vulnerability Scan
+
+on:
+  schedule:
+    # Run once every day
+    - cron: '0 0 0 * *'
+
+jobs:
+  scan:
+    name: Vulnerability scan
+    runs-on: ubuntu-latest
+
+    env:
+      DOCKERFILE: Dockerfile_with_poetry_lock
+
+    steps:
+    - name: Checkout git repository 🕝
+      uses: actions/checkout@v2
+
+    - name: Get latest tagged Rasa Open Source version 🔎
+      # We only scan the latest stable version of the Rasa Open Source Image
+      run: |
+        git fetch --depth=1 origin "+refs/tags/*:refs/tags/*"
```

Comment:
why should this run the tag and not just the master branch?

Reply:
You mean against `latest` (I'd like to avoid re-building the docker images) ? Can do, then we are dependent that changes from the patch branches are merged to master asap (which should be done anyway).

Reply:
yes I think that makes sense :+1: